src/Engine/ProtoCore/ExtendedLibraries/BuiltIns.ds

```diff
@@ -0,0 +1,64 @@
+class List extends DSCore.List
```

Comment:
That is already imported in the sequence of importing preloaded libraries but we can have the import statement here just in case the sequence is not followed.